src/main/java/com/stripe/param/PaymentIntentCreateParams.java

```diff
@@ -51,18 +43,6 @@
   @SerializedName("confirm")
   Boolean confirm;
 
-  /**
-   * One of `automatic` (default) or `manual`.
-   *
-   * <p>When the confirmation method is `automatic`, a PaymentIntent can be confirmed using a
-   * publishable key. After `next_action`s are handled, no additional confirmation is required to
-   * complete the payment.
-   *
-   * <p>When the confirmation method is `manual`, all payment attempts must be made using a secret
-   * key. The PaymentIntent returns to the `requires_confirmation` state after handling
-   * `next_action`s, and requires your server to initiate each payment attempt with an explicit
-   * confirmation.
-   */
```

Comment:
One of those "we lost the basic string documentation". Sigh.

Reply:
I guess this is because we're not documenting the individual enum values: https://stripe.com/docs/api/payment_intents/create#create_payment_intent-confirmation_method.

Not worth blocking this release, but we should come up with a better solution for client libs to document both the field and the possible values.